video_emotion/emotionTagger/face_emotion_extraction.py

```diff
@@ -4,6 +4,7 @@
 import numpy
 from keras.models import load_model
 import glob
+from filehandler.handle_path import get_absolute_path
```

Comment:
I think 'simple' imports should be grouped and then more 'advanced' imports with "from whatever import whatever2"